README.md

```diff
@@ -1,2 +1,29 @@
-# dry21-scripts
-Enapter Dryer DRY 2.1 control scripts examples 
+## Introduction
+
+Enapter Dryer is a hybrid temperature/pressure swing adsorption system increasing hydrogen purity to 99,999%. Find out more about [Enapter Dryer](https://handbook.enapter.com/electrolyser/dry21/dry21.html).
+
+We developed [Dryer Control Network](https://handbook.enapter.com/electrolyser/el21a/el21a.html#%F0%9F%95%B8%EF%B8%8F-dryer-control-network) - technology that allows you to control and monitor Enapter Dryer from [Electrolyser](https://handbook.enapter.com/electrolyser/el21a/el21a.html) Web GUI or Modbus TCP interfaces. Also when Dryer Control Network is enabled, Dryer starts automatically when at least one of connected electrolysers is steady.
```

Comment:
Maybe we change links to handbook using our [link shortener](https://go-admin.enapter.com)?

Reply:
Fixed.